driverkit/README.md

```diff
@@ -98,26 +98,41 @@ make -e TARGET_DISTRO=debian stats
 
 ## FAQ
 
-Q: Where can I found the list all pre-compiled drivers?
+### Q: Where can I found the list all pre-compiled drivers?
+
 A: Go to [https://download.falco.org/driver/site/index.html](https://download.falco.org/driver/site/index.html)
 
 ![drivers website screenshots](./drivers_website_screenshot.png)
 
-Q: Falco doesn't find the kernel module/ eBPF probe for my OS, what do I do?
-A: 
+### Q: Falco doesn't find the kernel module/ eBPF probe for my OS, what do I do?
+
+A: You can generate and contribute configurations for your OS, as follows:
+
+- [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) this [repository](https://github.com/falcosecurity/test-infra)
+- From the `driverkit` directory of the repository, run:
+  ```shell
+  make generate -e TARGET_DISTRO=<TARGET_DISTRO> -e TARGET_KERNEL=<TARGET_KERNEL>
+  ```
+  > Besides `$TARGET_DISTRO` and `$TARGET_KERNEL`, you can find more filters [here](#available-filters), that help you to target specific kernel releases.
+  >
+  > If you want the drivers for your local kernel, you can fill `$TARGET_KERNEL` with: `"$(uname -r)_$(uname -v | sed 's/#\([[:digit:]]\+\).*/\1/')"`.
+  >
+  > Available values for `$TARGET_DISTRO` can be found [here](https://github.com/falcosecurity/driverkit#supported-targets).
+
```

Comment:
I would also explain how to add "kernelurls" config key to the created conf file to make sure that driverkit is able to find and download them, if possible!

Reply:
The scope of this answer is a quick guide.
I'd instead put a detailed reference documentation about driverkit into its repo and linking to it here. WDYT @FedeDP?

Reply:
Yep, makes sense, thank you!